docs/source/gcp/examples/appengine.rst

```diff
@@ -0,0 +1,24 @@
+App Engine - Audit application changes
+=======================================
+Custodian can audit application changes (e.g. a new version has been deployed). Note that the ``notify`` action requires a Pub/Sub topic to be configured.
```

Comment:
this is another generic example that can be removed.

Reply:
Done.